src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -144,14 +140,17 @@ object DockerPlugin extends AutoPlugin {
         }
       },
       sourceDirectory := sourceDirectory.value / "docker",
-      stage := Stager.stage(Docker.name)(streams.value, stagingDirectory.value, mappings.value),
+      stage := {
+        dockerGenerateConfig.value
```

Comment:
I would rather declare this dependency with

```scala
stage := (stage dependsOn dockerGenerateConfig).value
```

I'm not sure if you have to scope the right hand side `stage` with `in Docker` or not. The `inConfig` semantics always confuse me.